bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/handler/BaseDeviceHandler.java

```diff
@@ -54,15 +53,8 @@ public BaseDeviceHandler(Thing thing) {
 
     @Override
     public void initialize() {
-        Bridge bridge = getBridge();
-
-        if (bridge == null) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
```

Comment:
I think this is still needed in case thing is configured without bridge, but in this case it should be offline with **CONFIGURATION_ERROR**. Please see #12331.

Reply:
I thought that @fwolter was saying that the framework takes care not to initialize the things if a bridge is a) offline, b) wrongly configured, or c) missing. Or??

Reply:
I don't remember any such statement about c) missing? Could be tested by simply creating a things file with a thing not assigned to any bridge. In the linked PR he did not remove the check for bridge being null.

Reply:
Ok.